tests/ct_out_of_memory_handling.c

```diff
@@ -170,14 +171,14 @@ void test_alloc_failure_handling(void) {
 
     // Connect
     {
-        for (int i = 0; i < 128; ++i) {
+        for (int i = 0; i < 148; ++i) {
```

Comment:
Because the stack-allocated iterator uses fewer allocations?

Reply:
I added the config to also parse replicas in this testcase, This required a couple of more allocations to succeed.